integration/cluster.go

```diff
@@ -409,3 +413,61 @@ func (c *testCluster) RotateRootCA(cert, key []byte) error {
 		return err
 	}, opsTimeout)
 }
+
+func (c *testCluster) RotateUnlockKey() error {
```

Comment:
This was to add tests to check if manager could lock/rotate/unlock in FIPS mode.